docs/eigenlayer/avs-guides/keys/_category_.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Let's change the path here to `key-management` instead of `keys`

Reply:
Done